modules/client/src/test/java/org/apache/ignite/client/fakes/FakeInternalTable.java

```diff
@@ -53,6 +53,8 @@
  * Fake internal table.
  */
 public class FakeInternalTable implements InternalTable {
+    public static final int PARTITIONS = 4;
```

Comment:
public?

Reply:
Yes, this is used to initialize `FakeCatalogService` and in some other places which should have the same number of partitions to make the tests work. The idea is to express this dependency explicitly and avoid magic numbers.

Reply:
The getter is not `static`.